fs/mmap/fs_rammap.h

```diff
@@ -53,29 +35,28 @@
  * - There are not access privileges.
  */
 
-struct fs_rammap_s
-{
-  struct fs_rammap_s *flink;       /* Implements a singly linked list */
-  FAR void           *addr;        /* Start of allocated memory */
-  size_t              length;      /* Length of region */
-  off_t               offset;      /* File offset */
-};
+#ifndef __FS_MMAP_FS_RAMMAP_H
```

Comment:
why not keep the file guard in the origin place?

Reply:
I just moved the comment to to top of the file, it was originally describing functionality of the structure which was removed. Now it describes the functionality of the whole driver